src/test/java/com/thealgorithms/datastructures/hashmap/hashing/MapTest.java

```diff
@@ -0,0 +1,47 @@
+package com.thealgorithms.datastructures.hashmap.hashing;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+abstract class MapTest {
+    abstract <Key extends Comparable<Key>, Value> Map<Key, Value> getMap();
+
+    @Test
+    void mapFunctionsTest() {
```

Comment:
Could you split it into multiple tests so that where each test is verifying one function?

Reply:
Everything else looks good

Reply:
Sure, done.